### PR TITLE
bump k8s libs

### PR DESCRIPTION
**Pull request by xh3b4sd**
Towards https://github.com/giantswarm/giantswarm/issues/2844.

### CODE REVIEW

**Comment by xh3b4sd** [comment]
Successfully tested on geckon. (besides the Vault issues which Tuomas is working on)

File: integration/operator_test.go [hunk 1]
```diff
@@ -109,7 +109,6 @@ func (cs *clients) tearDown() {
 	runCmd("helm delete cert-operator --purge")
 	runCmd("helm delete cert-resource-lab --purge")
 	cs.K8sCs.CoreV1().Namespaces().Delete("giantswarm", &metav1.DeleteOptions{})
-	cs.K8sCs.ExtensionsV1beta1().ThirdPartyResources().Delete("certificate.giantswarm.io", &metav1.DeleteOptions{})
```
Comment (MarcelMue) [inline integration/operator_test.go]: Why is this removed?

Comment (xh3b4sd) [reply integration/operator_test.go]: Because this is about TPRs, which do not exist anymore.
